config/install/apigee_kickstart_customizer.theme.apigee_kickstart.yml

```diff
@@ -3,6 +3,6 @@ values:
   '--ak-header-color': '#ffffff'
   '--ak-footer-color-bg': '#505050'
   '--ak-footer-color': '#fcfcfc'
-  '--ak-accent-color': '#2196f3'
+  '--ak-accent-color': '#0A5994'
```

Comment:
@giteshk and @raakesh-blokhra ,
Changing background color resolves most of the `Contrast Errors` accessibility issue.
Please let me know if this looks good to you.

Please check the sceenshot below.
**Before**
![Screenshot 2022-09-06 8 44 35 PM](https://user-images.githubusercontent.com/75600200/188672965-d5df810c-5d5b-4d1f-b0d3-ce313622b78a.png)

**After**
![Screenshot 2022-09-06 8 44 14 PM](https://user-images.githubusercontent.com/75600200/188673014-e92abbda-c8c5-4577-9465-82c2acd2e6ae.png)